.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -117,7 +117,33 @@ jobs:
             
             mkdir ./KeySetCache
             KEY_CACHE_DIRECTORY=./KeySetCache make pytest
-            
+
+      - name: Test notebooks
+        uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # v3
+        if: ${{ !contains(inputs.instance_type, 'p3') }}
+        with:
+          registry: ghcr.io
+          image: ${{ env.DOCKER_IMAGE_TEST }}
+          username: ${{ secrets.GHCR_LOGIN }}
+          password: ${{ secrets.GHCR_PASSWORD }}
+          options: >-
+            -v ${{ github.workspace }}:/concrete
+            -v ${{ github.workspace }}/build:/build
+          shell: bash
+          run: |
+            set -e
+
+            dnf -y install graphviz graphviz-devel
+
+            cd /concrete/frontends/concrete-python
+            make venv
+            source .venv/bin/activate
+
+            export COMPILER_BUILD_DIRECTORY=/build
+
+            make test-notebooks
```

Comment:
I copied the block before for pytest; it's ok?

Reply:
same as macos, you should create a step to prepare the test env

Reply:
Nope, we're doing things slightly differently, this should be okay. That being said, maybe just run `make test-notebooks` right after `make pytest`, it would reduce the duplication of the utility commands.